scrypto/src/runtime/runtime.rs

```diff
@@ -150,4 +150,9 @@ impl Runtime {
     pub fn fee_balance() -> Decimal {
         ScryptoEnv.fee_balance().unwrap()
     }
+
+    pub fn panic(message: String) -> ! {
+        ScryptoEnv.panic(message).unwrap();
```

Comment:
Can we instead have ScryptoEnv.panic return `!` so that we don't have this weird loop?

Reply:
Responded in another comment